pkg/resources/schema.go

```diff
@@ -79,16 +101,14 @@ func Schema() *schema.Resource {
 		SchemaVersion: 1,
 
 		CreateContext: CreateContextSchema,
-		ReadContext:   ReadContextSchema,
+		ReadContext:   ReadContextSchema(true),
 		UpdateContext: UpdateContextSchema,
 		DeleteContext: DeleteContextSchema,
+		Description:   "Resource used to manage schema objects. For more information, check [schema documentation](https://docs.snowflake.com/en/sql-reference/sql/create-schema).",
 
 		CustomizeDiff: customdiff.All(
```

Comment:
I'm wondering if we should somehow detect change to recompute the Describe output? Because right now I think it will be always the same and never change (because of lacking custom diff to recompute). Btw, you could at least add ComputedIfAnyAttributeChanged for Describe output with "name" attrib.

Reply:
Well, in description we only have a list of objects inside the schema, and we're not specifying any inside terraform config. Added ComputedIfAnyAttributeChanged for Describe output with "name", though.